Perl 5.40 warns when calling import on a package that does not define it.
My::Builder uses warnings FATAL => 'all', so the versioned
`use Module::Build '0.36'` form aborts the build.

https://bugs.gentoo.org/934245
https://bugs.debian.org/1078039

--- a/inc/My/Builder.pm
+++ b/inc/My/Builder.pm
@@ -8,7 +8,7 @@
 use File::Slurp qw(read_file write_file);
 use File::Spec::Functions qw(catdir catfile rootdir);
 use IO::File qw();
-use Module::Build '0.36' => qw();
+use Module::Build qw();
 use autodie qw(:all move read_file write_file);
 use parent 'Module::Build';
 use Locale::Maketext::Extract;
